developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -3,49 +3,47 @@ title: "System Integrators Guide"
 slug: "system-integrators-guide"
 ---
 
-:::tip 
-This documentation is currently under construction with more being added on a regular basis.
-:::
-
-If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform. This guide assumes that you are familiar with the blockchains.
+If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform.
 
 ## Overview
 
-This guide will overview the following topics for integrating with Aptos:
-- Preparing an environment for testing.
-- Create an account on the blockchain.
-- Exchange account identifiers with another entity on the blockchain, for example, to perform swaps.
-- Create a transaction.
-- Obtain a gas estimate and validate the transaction for correctness.
-- Submit the transaction to the blockchain.
-- Wait for the outcome of the transaction.
-- Query historical transactions and interactions for a given account with a specific account, i.e., withdraws and deposits.
+This document will guide you through the following tasks to integrate with Aptos:
+1. Prepare an environment for testing.
+1. Create an account on the blockchain.
+1. Exchange account identifiers with another entity on the blockchain, for example, to perform swaps.
+1. Create a transaction.
+1. Obtain a gas estimate and validate the transaction for correctness.
```

Comment:
Only annoying thing about the `1.`s is when you're not reading it rendered, but it's definitely easier to add more this way (without having to update all the numbers when you insert int he middle)

Reply:
I can revert. Do most people view the Markdown raw? We should optimize for the majority...